cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/AwsObjectStorageConnectorTest.java

```diff
@@ -0,0 +1,76 @@
+package com.sequenceiq.cloudbreak.cloud.aws;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.AmazonS3Exception;
+import com.sequenceiq.cloudbreak.cloud.aws.view.AwsCredentialView;
+import com.sequenceiq.cloudbreak.cloud.exception.CloudConnectorException;
+import com.sequenceiq.cloudbreak.cloud.model.objectstorage.ObjectStorageMetadataRequest;
+import com.sequenceiq.cloudbreak.cloud.model.objectstorage.ObjectStorageMetadataResponse;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
please use the new JUnit everywhere

Reply:
cloud-aws module is using JUnit4 at the moment. I do not wish to transform all of the tests.